app/src/main/java/com/android/unio/model/event/Event.kt

```diff
@@ -33,5 +33,5 @@ data class Event(
     val price: Double = 0.0,
     val date: Timestamp = Timestamp(Date()),
     val location: Location = Location(),
-    val types: List<EventType> = mutableListOf()
+    val types: List<EventType> = mutableListOf<EventType>(EventType.JAM)
```

Comment:
You are right, I added this just because otherwise the app would crash. We should consider refactor by removing default value or handle cases where no types where given